### DISCUSSION TITLE
我看起来很像人机吗？

### DISCUSSION THREAD

**Discussion by Aha-SWYH** [share121/inter-knot#2851] [其他]
![Screenshot_2024_0818_222642](https://github.com/user-attachments/assets/c7477f0e-2f12-4969-9072-8a219ac620a9)
我放图了，不知道能不能看到。

我在搞那个米游社的表情包，结果有一个验证一直过不了 求解决方法

**Comment by Aries-Woof** [comment]
翻墙吧，速度快一点<img src="https://bbs-static.miyoushe.com/static/2024/03/21/f5f9f9902140f168d5b804fe6c365ea6_4229255832036354153.png" alt="丽都漫步-哲" width="75" >

**Comment by Dave-12138** [comment]
其实……国内IP访问没有人机验证

**Reply by Dave-12138** [reply]
也就是说，关掉代理就没有人机验证了

**Comment by fairy-fire** [comment]
俺也一样
![Uploading Screenshot_20240819_133824.jpg…]()
